go.mod

```diff
@@ -8,23 +8,23 @@ require (
 	github.com/gorilla/websocket v1.5.3
 	github.com/natefinch/lumberjack v2.0.0+incompatible
 	github.com/stretchr/testify v1.8.0
+	golang.org/x/text v0.21.0
+	gopkg.in/yaml.v3 v3.0.1
```

Comment:
If we're adding `yaml.v3` we should make sure all instances of `yaml.v2` are replaced with `yaml.v3` and `yaml.v2` is removed.

Reply:
Good point, will update later.

Reply:
Also need to update Volte6/ansitags#9

Reply:
`yaml.v3` has some breaking changes: go-yaml/yaml#661